### PR TITLE
GDScript: Fix warning ignoring for member variables

### LINKED ISSUE #72135: @warning_ignore(unused_private_class_variable) ignores all following matching statements. Not just the first matching statement
### Godot version

v4.0.beta15.official [4fa6edc88]

### System information

Windows 11

### Issue description

The documentation for `@warning_ignore` states:

> Mark the following statement to ignore the specified warning

In a situation like this all three `unused_private_class_variable` warnings will be ignored though:

```
# some_script.gd
extends Node

@warning_ignore(unused_private_class_variable)
var _v1
var _v2
var _v3

```

### Steps to reproduce

1. Have multiple unused private variables in a script.
2. Place @warning_ignore(unused_private_class_variable) above the statements (play with where you put it) and see how always all matching statements below are ignored.

### Minimal reproduction project

[bug_warning_ignore_ignores_multiple_warnings_at_once.zip](https://github.com/godotengine/godot/files/10512675/bug_warning_ignore_ignores_multiple_warnings_at_once.zip)

### PR DESCRIPTION
**Pull request by dalexeev**
Closes #72135.

### CODE REVIEW

File: modules/gdscript/tests/scripts/analyzer/warnings/unused_private_class_variable.gd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# GH-72135
+
+var _a
+@warning_ignore("unused_private_class_variable")
+var _b
+@warning_ignore("unused_private_class_variable") var _c
```
Comment (anvilfolk) [inline modules/gdscript/tests/scripts/analyzer/warnings/unused_private_class_variable.gd:6]: Was the lack of a newline here intentional, to make sure the annotation applied to `_c`? :)

**Review by anvilfolk — APPROVED** [review]
Looks like something was inside a bracket when it shouldn't have been. Didn't dive too deep into the code, but with the tests it seems to check out just fine!

File: modules/gdscript/tests/scripts/analyzer/warnings/unused_private_class_variable.gd (hunk 1 above)
Comment (dalexeev) [reply modules/gdscript/tests/scripts/analyzer/warnings/unused_private_class_variable.gd:6]: Yes, I added this case for reinsurance. I can remove it if you think it's not needed.

Comment (anvilfolk) [reply modules/gdscript/tests/scripts/analyzer/warnings/unused_private_class_variable.gd:6]: I think it's fine! :)

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.4.

**Pull request merged**